### PR TITLE
solve label error when image is shuffled

### PR DESCRIPTION
**Pull request by billzyx**
If I do as follows, `to_categorical()` will return wrong labels because data is shuffled. So I need labels to be shuffled, too.
```python
from keras.preprocessing.image import ImageDataGenerator
from keras.utils.np_utils import to_categorical

datagen = ImageDataGenerator()
generator = datagen.flow_from_directory(
        train_data_dir,
        target_size=(img_width, img_height),
        batch_size=batch_size,
        class_mode=None,
        shuffle=True,
        seed=seed)
bottleneck_features_train = model.predict_generator(
     generator, (generator.n // batch_size)+0)
bottleneck_features_train_lable = to_categorical(generator.classes, num_classes=None)
```

### CODE REVIEW

**Comment by fchollet** [comment]
What is this meant to achieve?

**Comment by fchollet** [comment]
Thanks for editing your post to make your use case clear. The proper solution here is not to use `shuffle` in `predict_generator`.
